versioned_docs/version-v6.0.0/framework/pubsubV2.md

```diff
@@ -72,41 +118,64 @@ Once a message has been received, it is then deferred so that the deferred messa
     ...
 ]
 ```
+</details>
+
+:::warning[internal workaround]
+Because messages can be 'acknowledged' separately from the location where it is received by 'subscription', the message is internally [deferred](https://learn.microsoft.com/en-us/azure/service-bus-messaging/message-deferral). This is due to restrictions in the Azure SDK which impose that a message can only be settled by the same receiver instance which received the message. Deferring a message allows it to be picked up by any receiver.
+:::
 
+## ✔️ Acknowledge message
+The `/api/Acknowledge` endpoint allows users to 'settle' a previously received message via the [`/api/Subscribe` endpoint](#️-subscribe-for-messages). The **sequence number** of the message is required to run this operation.
 
-## Acknowledge
+The following request properties are available during acknowledgement:
```

Comment:
Might be better to rename to The following request properties can be supplied?

Currently it seems a bit like these properties are available but we actually need to supply them ourselves

Reply:
Made it `must/can be supplied` as otherwise, the `can` is also invalid in case of required properties.
Changes made here for all: https://github.com/invictus-integration/docs-ifa/pull/378/commits/776b21304079138bd7984f1fc3c9c0049e69f354